docs/topics/periodic-stats.rst

```diff
@@ -0,0 +1,298 @@
+    :type only_export_on_change: bool
+
+    :param only_export_on_close: If active, value will be only generated when the spider is closed.
+    :type only_export_on_close: bool
+
+You can define many observers::
+
+    PERIODIC_STATS_OBSERVERS = [
+        stats.Observer(key='downloader/request_count'),
+        stats.Observer(key='downloader/response_count'),
+        stats.Observer(key='httpcache/hit'),
+    ]
+
+    """
+    Generated stats:
+    {'downloader/request_count': 10, 'downloader/request_count': 10, 'httpcache/hit': 0}  # for t=1
+    {'downloader/request_count': 23, 'downloader/request_count': 20, 'httpcache/hit': 3}  # for t=2
+    {'downloader/request_count': 23, 'downloader/request_count': 20, 'httpcache/hit': 3}  # for t=3
+    {'downloader/request_count': 34, 'downloader/request_count': 28, 'httpcache/hit': 7}  # for t=4
+    ...
+    """
+
+To use a different name for the stats value use the ``export_key`` parameter::
+
+    PERIODIC_STATS_OBSERVERS = [
+        stats.Observer(key='downloader/request_count', export_key='requests'),
+        stats.Observer(key='downloader/response_count', export_key='responses'),
+        stats.Observer(key='httpcache/hit', export_key='from_cache'),
+    ]
+
+    """
+    Generated stats:
+    {'requests': 10, 'responses': 10, 'from_cache': 0}   # for t=1
+    {'requests': 23, 'responses': 20, 'from_cache': 3}   # for t=2
+    {'requests': 23, 'responses': 20, 'from_cache': 3}   # for t=3
+    {'requests': 34, 'responses': 28, 'from_cache': 7}   # for t=4
+    ...
+    """
+
+You can use several observers at the same key, but then ``export_key`` can't be repeated::
+
+    PERIODIC_STATS_OBSERVERS = [
+        stats.Observer(key='downloader/request_count', export_key='requests_counter'),
+        stats.Observer(key='downloader/request_count', export_key='another_request_counter'),
+    ]
+
+    """
+    Generated stats:
+    {'requests_counter': 10, 'another_request_counter': 10}  # for t=1
+    {'requests_counter': 23, 'another_request_counter': 23}  # for t=2
+    {'requests_counter': 23, 'another_request_counter': 23}  # for t=3
+    {'requests_counter': 34, 'another_request_counter': 34}  # for t=4
+    ...
+    """
+
+By default values are accumulated, but we can also use just differences between intervals with the
+``use_partial_values`` parameter::
+
+    PERIODIC_STATS_OBSERVERS = [
+        stats.Observer(key='downloader/request_count', export_key='requests', use_partial_values=True),
+    ]
+
+    """
+    Generated stats:
+    {'requests': 10}  # for t=1
+    {'requests': 13}  # for t=2
+    {'requests': 0}   # for t=3
+    {'requests': 11}  # for t=4
+    ...
+    """
+
+To minimize generated data we can choose to export data only when it changes with the
+``only_export_on_change`` parameter::
+
+    PERIODIC_STATS_OBSERVERS = [
+        stats.Observer(key='downloader/request_count', export_key='requests', only_export_on_change=True),
```

Comment:
They're different things, you can combine them.
`only_export_on_change=True` will only export value if it has changed since last export.
`use_partial_values =True` will calculate the value difference since last export, by default (False) it will export the accumulated value.